pmml_recommendation/recommender-engine/pom.xml

```diff
@@ -0,0 +1,79 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>org.kie</groupId>
+    <artifactId>recommender-engine</artifactId>
+    <name>PMML Recommendater Engine</name>
+    <version>1.0-SNAPSHOT</version>
+
+    <properties>
+        <org.kie.version>7.56.0-SNAPSHOT</org.kie.version>
```

Comment:
Please remember to update to final when released

Reply:
@danielezonca :+1: